community/views.py

```diff
@@ -1,17 +1,106 @@
 from django.shortcuts import render
-
 import json
 
 from django.http import JsonResponse
 from django.views import View
 
-from .models import PostImage, Post, Hashtag
+from .models import HouseSize, HouseStyle, HousingType, Space,  Post, PostBlock
+#from user.utils import login_decorator
+from product.models import *
+from user.models import *
+
+class Posting(View):
+
+    #@login_decorator
+    def post(self, request):
+        data = json.loads(request.body)
+
+        # 토큰 전까지 사용할  유저 아이디
+        user_id      = 2
+
+        house_size   = data.get('house_size')
+        house_style  = data.get('house_style')
+        housing_type = data.get('housing_type')
+        block        = data['block']
+
+        try:
+            Post.objects.create(
+                user_id=user_id,
+                house_size_id=house_size,
+                house_style_id=house_style,
+                housing_type_id=housing_type
+            )
+
+            post = Post.objects.all().last()
```

Comment:
```suggestion
           post = Post.objects.create(
                user_id=user_id,
                house_size_id=house_size,
                house_style_id=house_style,
                housing_type_id=housing_type
            )
```
최근 생성된걸 관리하려면 바로 담으면 가능합니다.